website/docs/r/datapipeline.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "aws"
+page_title: "AWS: aws_datapipeline"
+sidebar_current: "docs-aws-resource-datapipeline"
+description: |-
+  Provides a AWS DataPipeline.
+---
+
+# aws_datapipeline
+
+Provides a Data Pipeline resource.
+
+## Example Usage
+
+### Create Pipeline Only
```

Comment:
For single documentation examples, we typically omit any additional headers. In this case, the resource can also support updates of tags and deletion of itself, so we should remove this to prevent any confusion as well.

```suggestion
```

Reply:
fixed